Core/Services/StitchCoreRemoteMongoDBService/Sources/StitchCoreRemoteMongoDBService/Sync/DataSynchronizer.swift

```diff
@@ -416,6 +550,7 @@ public class DataSynchronizer: NetworkStateDelegate, FatalErrorListener {
     }
 
 
+    typealias DeferredBlock = (() -> Void)
```

Comment:
[nit] Not sure this type is complex enough to warrant a typealias

Reply:
I did it because `let deferredEventEmittingBlock: (() -> Void)? = nil` is a little weird to read but I could change it back